geode-wan/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommandDUnitTest.java

```diff
@@ -33,6 +37,7 @@
 import org.apache.geode.test.junit.rules.serializable.SerializableTestName;
 
 @Category({WanTest.class})
+@RunWith(JUnitParamsRunner.class)
```

Comment:
Please use `GeodeParamsRunner` instead. `JUnitParamsRunner` is incompatible with JUnit Platform, which we now use to run tests.

`GeodeParamsRunner` extends `JUnitParamsRunner`, so you can use it the same way in tests. But it overrides `JUnitParamsRunner` to make it more compatible with JUnit Platform.

Reply:
Thank you! I have made the change